tests/webgl/test_remote_save_load.py

```diff
@@ -0,0 +1,73 @@
+"""
+The following instruction is based on web/README.md.
+
+Setup an RPC server:
+$ python -m tvm.exec.rpc_proxy --example-rpc=1
+
+Go to http://localhost:9190 in browser.
+
+Click "Connect To Proxy".
+
+Run this test script:
+$ python tests/webgl/test_remote_save_load.py
+"""
+
+import numpy as np
+import tvm
+from tvm.contrib import rpc, util, emscripten
+
+proxy_host = "localhost"
+proxy_port = 9090
+
+def test_remote_save_load():
```

Comment:
Make testcase compatible with nose-test. Change name of non-nose testable function to be other-name, like try_remote_save_load.

Always detect main when run the test function, so it won't get invoked by nose
 a flag "__main__", and

Reply:
Changed.